apps/portals/src/configurations/elportal/synapseConfigs/cohortbuilder.ts

```diff
@@ -12,9 +12,62 @@ import {
   handleFilesToParticipants,
   handleSelectedFilesToParticipants,
 } from './handleFilesToParticipants'
+import {
+  QueryWrapperSynapsePlotProps,
+  QueryWrapperSynapsePlotRowClickEvent,
+} from 'synapse-react-client/src/components/QueryWrapperPlotNav/QueryWrapperSynapsePlot'
+import { SynapseUtilityFunctions } from 'synapse-react-client'
 
 const rgbIndex = 1
 
+const handlePlotClick = (event: QueryWrapperSynapsePlotRowClickEvent) => {
+  // window.alert(`Handling click on this row: ${JSON.stringify(event.row)}`)
+  const { setRangeFacetValue } = event.queryContext
+  let newMin: string | undefined = undefined,
+    newMax: string | undefined = undefined
+  const value = event.row.values[0] as string
+  const values = value.match(/\d+/g)
+  if (values && values.length > 0) {
+    newMin = values[0]
+    if (values.length > 1) {
+      newMax = values[1]
+    }
+  }
+  // need to set both for the combined range filter
+  setRangeFacetValue({ columnName: 'minAge' }, newMin, newMax)
+  setRangeFacetValue({ columnName: 'maxAge' }, newMin, newMax)
+}
+const getPlotConfig = (tableId: string) => {
```

Comment:
ELITE portal (complex) custom interactive plot definition, with custom click handler.  May want to review this last!